x/oracle/ante.go

```diff
@@ -47,6 +49,31 @@ func (spd SpammingPreventionDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, si
 	return next(ctx, tx, simulate)
 }
 
+func (spd SpammingPreventionDecorator) AnteDeps(txDeps []sdkacltypes.AccessOperation, tx sdk.Tx, next sdk.AnteDepGenerator) (newTxDeps []sdkacltypes.AccessOperation, err error) {
```

Comment:
are these AnteDeps functions going to look similar across all antes (i.e. oracle, gasless)? If so, maybe it'll make sense to have this somewhere centralized. If not, ffti

Reply:
Tony did that in his PR - I'm going to remove my changes and rebase on his